scripts/http-form-brute.nse

```diff
@@ -315,6 +315,9 @@ end
 
 -- TODO: expire cookies
 local function update_cookies (old, new)
+  if not old then
```

Comment:
But the change below to the `while` loop condition should ensure `update_cookies` is never called if there was a socket error. We should take one of 2 options:

1. Don't check whether `old` is nil, creating a script crash if it ever is, so that we can more closely investigate the cause. I don't like this option because crashing is bad.
2. `if not old then return end` to avoid looping through, since this function is meant to modify an existing table. As it is, when `old` goes out of scope it is garbage-collected so all that work is pointless. We could add a debug statement to let us know if this condition is occurring.